src/http-spec.ts

```diff
@@ -73,9 +73,8 @@ export interface IHttpParam {
   name: string;
   style: HttpParamStyles;
   content?: IHttpContent;
-  description?: string;
```

Comment:
I think we can actually leave description as is (see https://gist.github.com/marbemac/1cd45ca27846f2b838a3ab369abc32cd#file-httpop-json-L35).

Reply:
Restored. The schema could still have a description though, is it ok to have two of them?